### ISSUE TITLE
MA Ermor issues

### ISSUE THREAD

**Issue by ConcernedSkeleton** [sum1won/summod#59]
That's it. I'm sick of all this "MA Ermor OP" bullshit that's going on in the /domg/ right now. Ermor deserves much better than that. Much, much better than that.
I should know what I'm talking about. I myself won 24 games as Ermor (that's around 86% win rate) and have been practicing with it for almost 2 years now. I can even rush people with rainbow pretenders now.
/domg/ (and sum1won in particular) spent years shitting on MA Ermor balance to produce the weakest MA nation known to Dominions 4 fans.
Skeleton is thrice as strong as flesh and thrice as hard for that matter too. Anything a living man can do, a skeleton can do even better. I'm pretty sure a skeleton could easily bisect a knight wearing full plate with a simple vertical slash.
Ever wonder why medieval people buried their dead deep in the ground? That's right, they were too scared to fight fearsome skeletons and their shadowy masters. Even in World War II, American soldiers targeted skeletons first because their killing power was feared and respected.
So what am I saying? Skeletons are simply the best type of unit that the world has ever seen, and thus, require better stats in the Dominions 4. Here is the stat block I propose for skeletons:
Hit points 48
Protection 16
Magic resistance 18
Strength 24
Attack skill 18
Defence sill 22
Precision 14
3 attacks (sword, skull, bones)
Fear 6
Invulnerability 15
Now that seems a lot more representative of the power of skeletons in real life, don't you think?
tl;dr = Skeletons need to do more damage in Dominions 4, see my new stat block.
